src/main/scala/mesosphere/marathon/core/launcher/impl/TaskOpFactoryHelper.scala

```diff
@@ -36,6 +46,6 @@ class TaskOpFactoryHelper(
       offerOperationFactory.reserve(frameworkId, newTask.taskId, resources),
       offerOperationFactory.createVolumes(frameworkId, newTask.taskId, localVolumes))
 
-    TaskOp.ReserveAndCreateVolumes(newTask, resources, localVolumes, oldTask, createOperations)
+    TaskOp.ReserveAndCreateVolumes(newTask, resources, localVolumes, createOperations)
```

Comment:
jepp, you should be right, no old task should exist for this. Good to make that explicit :+1: